File: mobile/src/main/java/com/keepnotes/ui/BaseActivity.java
package com.keepnotes.ui;

import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

public class BaseActivity extends AppCompatActivity {
    public boolean isActivityVisible = false;

    public void setActionBar(Toolbar toolbar, boolean homeAsUp, boolean homeEnabled, String title) {

        setSupportActionBar(toolbar);

        ActionBar actionBar = getSupportActionBar();
        if (actionBar != null) {
            actionBar.setDisplayHomeAsUpEnabled(homeAsUp);
            actionBar.setHomeButtonEnabled(homeEnabled);
            if (title == null) {
                actionBar.setDisplayShowTitleEnabled(false);
            }

            //if (!TextUtils.isEmpty(title)) {
            actionBar.setTitle(title);
            //}
        }

    }

    @Override
    protected void onResume() {
        super.onResume();
        isActivityVisible = true;
    }

    @Override
    protected void onPause() {
        isActivityVisible = false;
        super.onPause();
    }
}

File: mobile/src/main/java/com/keepnotes/ui/BaseActivity.kt
package com.keepnotes.ui

import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.Toolbar

open class BaseActivity : AppCompatActivity() {
    var isActivityVisible = false

    fun setActionBar(toolbar: Toolbar, homeAsUp: Boolean, homeEnabled: Boolean, title: String?) {

        setSupportActionBar(toolbar)

        val actionBar = supportActionBar
        if (actionBar != null) {
            actionBar.setDisplayHomeAsUpEnabled(homeAsUp)
            actionBar.setHomeButtonEnabled(homeEnabled)
            if (title == null) {
                actionBar.setDisplayShowTitleEnabled(false)
            }

            //if (!TextUtils.isEmpty(title)) {
            actionBar.title = title
            //}
        }

    }

    override fun onResume() {
        super.onResume()
        isActivityVisible = true
    }

    override fun onPause() {
        isActivityVisible = false
        super.onPause()
    }
}